api/base/pagination.py

```diff
@@ -21,6 +22,10 @@ class JSONAPIPagination(pagination.PageNumberPagination):
 
     page_size_query_param = 'page[size]'
 
+    def __init__(self, max_page_size=None, **kwargs):
+        super(JSONAPIPagination, self).__init__(**kwargs)
+        self.max_page_size = max_page_size or MAX_PAGE_SIZE
```

Comment:
Shouldn't this be set as a class variable? That's what it looks like in the docs: http://www.django-rest-framework.org/api-guide/pagination/#modifying-the-pagination-style . It seems preferable to do that rather than overriding `__init__` and changing its signature in a possibly breaking way.